rhel/vex/parser.go

```diff
@@ -331,9 +352,59 @@ func (c *creator) lookupVulnerability(vulnKey string, protoVulnFunc func() *clai
 	return &c.fixedVulns[idx], ok
 }
 
+type ranger struct {
+	lowest map[string]*claircore.Range
+}
+
+func newRanger() *ranger {
+	return &ranger{
+		lowest: map[string]*claircore.Range{},
+	}
+}
+
+func (r *ranger) add(packageName, fixedInVersion string) (*claircore.Range, error) {
```

Comment:
what exactly is this doing? I'm a bit confused about how the fixed ones are handled

Reply:
I added a the comment 
```
// Add takes a packageName and a fixedInVersion and returns a *claircore.Range relating
// to the fixedInVersion. Add also saves the lowest range per packageName that it's seen.
// This allows zeroLowest() to zero out the lowest values we saw per packageName.
```